### ISSUE TITLE
#847815517------------- Elements must have sufficient color contrast

### ISSUE THREAD

**Issue by khyatit13** [khyatit13/Coding-practice#589]
<strong>Source:</strong>`<span class="CalciteA-sc-1pkt4xd-3 StyledTopNavLink-sc-10bmdec-5 czDMUZ dtIewz sc-kngDgl kWgnVs">Review</span>`
<strong>Element Location: </strong>.dtIewz
<strong>Impact: </strong>serious
<strong>failureSummary:</strong> Fix any of the following:
  Element has insufficient color contrast of 3.71 (foreground color: #808080, background color: #f8f8f8, font size: 12.0pt (16px), font weight: normal). Expected contrast ratio of 4.5:1
Tags: cat.color,wcag2aa,wcag143,ACT
Screenshot:`C:\Users\khy12817\Desktop\Test.PNG`
